molecule/command/base.py

```diff
@@ -122,6 +124,33 @@ def execute_cmdline_scenarios(scenario_name, args, command_args, ansible_args=()
                 raise
 
 
+def _output_for_ci(func):
+    is_travis = os.getenv("TRAVIS") and os.getenv("CI")
+    if not is_travis:
+        return func
+
+    @functools.wraps(func)
+    def travis_wrapper(config, subcommand):
+        print(
+            (
+                "travis_fold:start:{0}.{1}"
+                "{2}Molecule --> Scenario: '{0}' --> Action: '{1}'{3}"
+            ).format(
+                config.scenario.name,
+                subcommand,
+                colorama.Style.BRIGHT + colorama.Fore.CYAN,
+                colorama.Style.RESET_ALL,
+            )
+        )
+        try:
+            return func(config, subcommand)
```

Comment:
I missed this return here. Added. Please try again.

Reply:
For background, I discovered that I missed this in the 2.22 branch, but didn't check to see if it was also required for 3.x. Oops.
See: https://github.com/cognifloyd/molecule/commit/6ddc9b2717c43a89bacf24ef7d4a04b498d0fe90